al2/x86_64/standard/3.0/Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN set -ex \
     && rpm --import https://download.mono-project.com/repo/xamarin.gpg \
     && curl https://download.mono-project.com/repo/centos7-stable.repo | tee /etc/yum.repos.d/mono-centos7-stable.repo \
     && amazon-linux-extras enable corretto8 \
+    && amazon-linux-extras enable docker \
```

Comment:
Why is this needed?

Reply:
The credential helper is available as part of the `docker` extra.  It's not included in the core yum repository.

Reply:
Thanks Samuel!